### ISSUE TITLE
how to construct the training set, verification set and test set 

### ISSUE THREAD

**Issue by 15605182960** [THUDM/GATNE#117]
Hello!
I would like to ask how to construct the training set, verification set and test set in the article code. Is there a specific code or idea?
